README.md

```diff
@@ -55,6 +55,18 @@ self.addEventListener('periodicsync', event => {
 });
 ```
 
+## Removing a periodic sync
```

Comment:
It would be good to add some context about _why_ you'd want to do this. Adding "when the user signs out" might provide that.